### ISSUE TITLE
[Bug]: The 'Date and time' controls cannot be expanded with voice access commands.

### ISSUE THREAD

**Issue by tarun29061990** [microsoft/fluentui#30342]
### Library

React Components / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: macOS 14.2.1
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 1.02 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Browsers:
    Chrome: 120.0.6099.234
    Edge: 120.0.2210.144
    Safari: 17.2.1
```

### Are you reporting Accessibility issue?

None

### Reproduction

No

### Bug Description

Prerequisites: 
Turn on the Voice Access from the system settings 
 
Repro steps:​ 
1.Open the Teams2.1 application 
2.Activate the calendar controls in the left navigation region. 
3.Activate the new meeting controls. 
4. It will open the new meeting screen, 
5, Give command as 'Show Numbers' & navigate to the date picker, time picker with Voice Access, and observe the issue. 
 
Actual Result:   
By using voice access command focus lands on the Date and time picker controls but Date and time picker controls are not getting expanded.
   
Expected Result:  
The Date and time picker controls should be expanded voice access command on the time and date edit fields. 
 
User Impact:  
Voice Access-dependent users will face difficulty in expanding date and time picker controls using voice commands. This limitation can hinder their ability to interact with the interface efficiently and a less satisfactory user experience.

### Requested priority

Normal

### Products/sites affected

MS teams

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by Hotell** [comment]
> Turn on the Voice Access from the system settings

from which system settings ?

**Comment by tudorpopams** [comment]
Hello @tarun29061990 , as @Hotell mentioned above, could you please share a codesandbox environment with us where this issue is isolated so we can make sure that it's something on our side which can be fixed?

**Comment by jurokapsiar** [comment]
Looking at DOM of the screen this appears to be a v8 issue
